share/locale/mscore_pl.ts

```diff
@@ -5508,7 +5508,7 @@ Domyślnie zostaną one umieszczone tak, aby ich prawy koniec znajdował się na
         <location filename="../../mscore/exportdialog.ui" line="122"/>
         <location filename="../../mscore/exportdialog.ui" line="198"/>
         <source>DPI</source>
-        <translation>DPI</translation>
+        <translation>dpi</translation>
```

Comment:
Sure about this? "DPI" is an acronym, not an abbreviation after all.

Reply:
Yes it is just like BPM, another acronym.
MuseScore Studio's translation on Transifex might also be wrong, as might the Wiki.
But I don't really care either way (I surely will keep it all caps in the German end British English trasnslations), just wanted to bring this up